test/integration_jenkins/plugin_dt_test.rb

```diff
@@ -51,47 +61,45 @@ def create_driver(conf = config)
 
   sub_test_case 'tests for #write' do
     test 'Write all records as a JSON array' do
-      nonce = (0...10).map { (65 + rand(26)).chr }.join
+      nonce = (0...10).map { rand(65..90).chr }.join
       puts "Generating random message: #{nonce}"
       d = create_driver
       d.run do
         d.feed('tag', event_time, { 'message' => nonce })
       end
 
       (0...40).each do |i|
-        puts "Getting logs attempt #{i+1}/40"
-
-        log = get_log(nonce)
-
-        break if log != nil
+        puts "Getting logs attempt #{i + 1}/40"
+        break if try_get_log(nonce)
 
         sleep 10
       end
 
-      puts 'got here'
+      raise 'Could not retrieve log after 40 attempts'
```

Comment:
The break above will only leave the loop, not the entire method, right?

Reply:
yes